driverconfig/config.yaml

```diff
@@ -6,6 +6,7 @@ supportedK8sVersions:
   - v121
   - v122
   - v123
+  - v124
```

Comment:
@shefali-malhotra We need to remove v121 if we are adding v124 here

Reply:
As these are common changes across drivers . So haven't changed it . As it might dependencies.